project/CONTRIBUTING.md

```diff
@@ -423,6 +435,17 @@ use the "Rebase and merge" feature.
 And finally, use your common sense : if you see a PR about a typo,
 or if there is a situation (faulty commit, revert needed) maybe you can merge it directly.
 
+#### Dependencies version dropping
+
+Do not merge any merge request dropping a dependency version support.
+Add the `RTM` label and specify it while asking for a release on Slack.
```

Comment:
what do you mean by "specify it" ?

Reply:
Add links to the needed merge on the release section.